lib/config/presets/gitea/index.ts

```diff
@@ -29,7 +29,7 @@ export async function fetchJSONFile(
   }
 
   // TODO: null check #22198
-  return parsePreset(fromBase64(res.content!));
+  return parsePreset(fromBase64(res.content!), fileName.includes('.json5'));
```

Comment:
I think we should pass the fileName instead of a bool. so we can maybe support yaml in future